torchtext/experimental/datasets/raw/language_modeling.py

```diff
@@ -16,37 +16,6 @@
 }
 
 
-class RawTextIterableDataset(torch.utils.data.IterableDataset):
-    """Defines an abstraction for raw text iterable datasets.
-    """
-
-    def __init__(self, iterator, start=0, num_lines=None):
```

Comment:
This is an API breaking change for the constructor, but then we're not exposing this class to users.